Eco.EM.Framework/Config/EMConfigureConfig.cs

```diff
@@ -68,5 +68,9 @@ public class EMConfigureConfig
         [LocDescription("Vehicle Configuration Values by modules. ANY change to this list will require a server restart to take effect.")]
         [LocDisplayName("Vehicle Configuration Values")]
         public SerializedSynchronizedCollection<VehicleModel> EMVehicles { get; set; } = new();
+
+        [LocDescription("Custom Configuration Values by modules. ANY change to this list will require a server restart to take effect.")]
+        [LocDisplayName("Custom Configuration Values")]
+        public SerializedSynchronizedCollection<CustomsModel> EMCustoms { get; set; } = new();
```

Comment:
Looks Good, however, given this is designed for custom data, maybe this should be in its own configuration setup,

we have EMConfigure

maybe another plugin called something like: EMConfigValues 

for this kinda of information, 

the em configure file gets massive which im already considering breaking up into smaller config files so having it as its own plugin might be a good way to go about it,

Reply:
ill approve the pr on successful build and then make the appropriate changes tonight to reflect the new config files